### PR TITLE
Add adapter option to keycloak.js

### PR DESCRIPTION
**Pull request by wtrocki**
Adding documentation for adapter option.

### CODE REVIEW

**Comment by wtrocki** [comment]
It seems that the build is failing due to missing link that is also broken on master.

File: securing_apps/topics/oidc/javascript-adapter.adoc [hunk 1]
```diff
@@ -227,6 +227,13 @@ responseMode::
 
 flow::
     Flow passed in init.
+    
+adapter::
+    Allows to override way how redirects and other browser related functionalities will be handled by library.
+    Possible options:
+    * "default" - using browser api for redirects (used by default)
+    * "cordova" - will be used automatically when library is working in cordova ecosystem. When specifying this option library will try to use InAppBrowser cordova plugin to load keycloak login/registration pages.
+    * custom - function that allows to implement custom adapter (only for advanced use cases)
```
Comment (matthewhelmke) [inline securing_apps/topics/oidc/javascript-adapter.adoc]: adapter::
    Allows you to override the way that redirects and other browser-related functions will be handled by the library.
    Available options:
    * "default" - the library uses the browser api for redirects (this is the default)
    * "cordova" - the library will try to use the InAppBrowser cordova plugin to load keycloak login/registration pages (this is used automatically when the library is working in a cordova ecosystem)
    * custom - allows you to implement a custom adapter (only for advanced use cases)

**Review by matthewhelmke — CHANGES_REQUESTED** [review]
Thank you. I have made some language edits during my review and posted them in a comment.

**Comment by wtrocki** [comment]
@matthewhelmke Thank you so much for reviewing that in so short notice.
I have applied your changes.

**Review by matthewhelmke — APPROVED** [review]

**Review by stianst — APPROVED** [review]

**Pull request merged**